README.md

```diff
@@ -6,7 +6,58 @@
 
 # SW Helpers
 
+## Contents
+
+- [Ways to Use the Libraries](#ways-to-use-the-libraries)
+- [Setting Up](#setting-up)
+- [The Libraries](#the-libraries)
+  - [sw-appcache-behavior](#sw-appcache-behavior)
+  - [sw-broadcast-cache-update](#sw-broadcast-cache-update)
+  - [sw-build](#sw-build)
+  - [sw-cache-expiration](#sw-cache-expiration)
+  - [sw-cli](#sw-cli)
+  - [sw-lib](#sw-lib)
+  - [sw-offline-google-analytics](#sw-offline-google-analytics)
+  - [sw-precaching](#sw-precaching)
+  - [sw-routing](#sw-routing)
+  - [sw-runtime-caching](#sw-runtime-caching)
+- [External Contributions](#external-contributions)
+- [License](#license)
+
+The sw-helpers library is ...
+
+## Ways to Use the Libraries
+
+Despite the number of modules in this package, there are a number of approaches to using these tools which many be used separately or together in any number of combinations.
+
+* **Basic caching strategies**: Use [sw-lib](.) to quickly implement one of the
+  [standard caching strategies](.).
+* **Build process**: Incorporate sw-helpers into your build process using
+  [sw-build](.).
+* **Command line**: Use a command line interface to generate a basic service
+  worker and play around with the features of the library.
+* **Go beyond the basics**: Implement more advanced capabilities and more
+  sophisticated use cases using any of the other libraries, which are described
+  below.
```

Comment:
I'd link straight to the section - "described below." confused me as the next step of setting up.